pkg/skaffold/config/util.go

```diff
@@ -257,10 +257,6 @@ func GetCluster(ctx context.Context, opts GetClusterOpts) (Cluster, error) {
 
 	var defaultRepo = opts.DefaultRepo
 
-	if defaultRepo.Value() != nil && cfg.DefaultRepo != "" {
-		defaultRepo = NewStringOrUndefined(&cfg.DefaultRepo)
-	}
-
 	if local && defaultRepo.Value() == nil {
```

Comment:
is `cfg.DefaultRepo` being checked somewhere else? Otherwise I think the author intended this to say:
```
if defaultRepo.Value() == nil && cfg.DefaultRepo != "" {
		defaultRepo = NewStringOrUndefined(&cfg.DefaultRepo)
}
```

Reply:
the reason I believe this piece of code is not needed is because typically when we call the following function:
https://github.com/GoogleContainerTools/skaffold/blob/1dbc7a2c8746201ccff73a54928a57d687eb020a/pkg/skaffold/deploy/util/util.go#L34-L52

We grab the value from the config and compare things then. This is how things usually worked before these few lines were added I believe

Reply:
sounds good